src/views/MainView/MainView.scss

```diff
@@ -19,24 +19,24 @@
     .LeftColumn {
       width: 280px;
       height: 100%;
-      transition: width 1.2s ease-out;
+      transition: width 0.4s ease-out;
```

Comment:
I see you did speed up the animation. That is good. I actually really think I should create a new landing page. But it is only an idea. I didn't put to much thought into it yet.